tests/chocolatey-tests/BundledApplications.Tests.ps1

```diff
@@ -6,6 +6,7 @@ Describe 'Ensuring correct version of <Name> is installed' -Tag BundledApplicati
 ) -Skip:(-not (Test-ChocolateyVersionEqualOrHigherThan "1.0.0")) {
     Context '<Name> is correctly installed' -Skip:(-not (Test-ChocolateyVersionEqualOrHigherThan $ChocolateyVersion)) {
         BeforeAll {
+            # Because we're not modifying the install in any way, there is no need to Initicalize-ChocolateyTestInstall

```

Comment:
```suggestion
            # Because we're not modifying the install in any way, there is no need to Initialize-ChocolateyTestInstall
```

Reply:
Thanks for catching that, but according to the git history this never happened 🤪